webpack/externals.js

```diff
@@ -4,6 +4,17 @@
  * @since 2.0.0
  */
 
+/**
+ * Convert any string to camelcase.
+ *
+ * @param string str String to convert
+ *
+ * @since 2.0.0
+ */
+function camalize(str) {
```

Comment:
The comment left by @tihomirselak is a valid one. `camalize` sounds wrong. And there are packages that are named `camelize` already.